### PR TITLE
Updates to support Odyssey DDR5

### PR DESCRIPTION
**Pull request by ruby452**
1.) Updates to chip-comb.xml to:
- support 2 men_ports per OCMB and 2 pervasive chip units (1 & 8)
- make ocmb be TYPE of OCMB_CHIP
- make temp sensors be TYPE of TEMP_SENSOR

2.) Updates to lcard-dimm-ddimm.xml to:
- update PMIC targets
- make temp sensors be TYPE of TEMP_SENSOR

3.) Updates to lcard-dimm-ddimm4u.xml:
- add 4 double top devices (0-3) with TYPE of POWER_IC
- remove ddr4 target and replace it with ddrA and ddrB targets
- add two mem_ports per OCMB and 2 pervasive chip units (1 & 8)
- add i2c-obmc-master target
- make temp sensors be TYPE of TEMP_SENSOR
- update PMIC targets

### CODE REVIEW

**Comment by ruby452** [comment]
Reverted TYPE of TEMP_SENSOR to NA since it's causing problems with the code

**Comment by dcrowell77** [comment]
> Reverted TYPE of TEMP_SENSOR to NA since it's causing problems with the code
That wasn't the issue.  The issue the <type> parameter, not the TYPE attribute.

```
<targetInstance>
	<id>i2c-ts0</id>
	<type>unit-i2c-slave</type>
```

vs

```
<targetInstance>
	<id>i2c-ts0</id>
	<type>temp_sensor</type>
```

File: parts/lcard-dimm-ddimm.xml [hunk 1]
```diff
@@ -321,10 +321,202 @@
...
+	<attribute>
+		<id>TYPE</id>
+		<default>PMIC</default>
+	</attribute>    
+</targetPart>
+<targetPart>
+	<id>i2c-slave</id>
+	<type>unit-i2c-slave</type>
+	<is_root>false</is_root>
+	<instance_name>i2c-slave</instance_name>
```
Comment (dcrowell77) [inline parts/lcard-dimm-ddimm.xml:346]: Isn't a name like "i2c-slave" too generic?  It seems like there will be some name collisions with a generic label like that.

**Review by dcrowell77 — APPROVED** [review]
Seems functionally correct

**Comment by ruby452** [comment]
I just push an update to this pending PR to fix the temp_sensors under the OCMB per feedback yesterday

**Review by dcrowell77 — APPROVED** [review]

**Review by adangster — APPROVED** [review]

**Pull request merged**